lib/features.json

```diff
@@ -17,11 +17,16 @@
   {
     "name": "velocityBlocksOnTop",
     "description": "Velocity changes are caused by the block the player is standing on",
-    "versions": ["1.15", "1.16"]
+    "versions": ["1.15", "1.17"]
   },
   {
     "name": "climbUsingJump",
     "description": "Entity can climb ladders and vines by pressing jump",
-    "versions": ["1.14", "1.15", "1.16"]
+    "versions": ["1.14", "1.15", "1.17"]
+  },
+  {
+    "name": "effectNamesAreRegistryNames",
+    "description": "Status effect names equal to their registry names",
+    "versions": ["1.17", "1.17"]
```

Comment:
duplicated

Reply:
isn't it [minVersion, maxVersion] like in mineflayer?

Reply:
I don't think so, but btw we should put this support feature thing in node-minecraft-data so it's the same everywhere

Reply:
alright, removed the duplicate